README.md

```diff
@@ -48,3 +48,24 @@ dependencies: [
 ```
 
 For detailed usage and API documentation, check [the documentation](https://swiftpackageindex.com/apple/swift-certificates/main/documentation/x509).
+
+## Benchmarks
+
+Benchmarks for `swift-certificates` are in a separate Swift Package in the `benchmarks` subfolder of this repository. 
```

Comment:
Oh I have one more nit: Can we name the folder `Benchmarks` to be in-line with `Sources` and `Tests`?

Reply:
its on purpose not uppercase as it is a separate package and folders of packages are lowercase.